src/Mailer/Email.php

```diff
@@ -411,25 +411,6 @@ public function getFrom()
         return $this->_from;
     }
 
-    /**
-     * From
-     *
-     * @deprecated 3.4.0 Use setFrom()/getFrom() instead.
-     * @param string|array|null $email Null to get, String with email,
-     *   Array with email as key, name as value or email as value (without name)
-     * @param string|null $name Name
-     * @return array|$this
-     * @throws \InvalidArgumentException
-     */
-    public function from($email = null, $name = null)
-    {
-        if ($email === null) {
-            return $this->getFrom();
-        }
-
-        return $this->setFrom($email, $name);
```

Comment:
@markstory we did not  have deprecation warning for `for()` method in 3.6

Reply:
@saeideng If we don't want add new deprecation message in patches release of 3.6 then can just add it for 3.7.